tutorials/firstcodemod.md

```diff
@@ -86,10 +86,10 @@ Right-click your project's "References" and select "Add Reference...", then set
 ## Recommended NuGet packages
 
 **We're recommending you to install the following NuGet packages:**
-- `Microsoft.Net.Compilers` - Maintain build compatibility on systems with older C# compilers installed.
+- `Microsoft.Net.Compilers` - Maintain build compatibility on systems with older C# compilers installed. **Latest version recommended**
 - `System.ValueTuple` - Used by HookedMethod.
 - `Mono.Cecil` - Used by HookedMethod and MonoMod. IL manipulation library. **0.10.0 or newer required**
-- `HookedMethod` - Hook Celeste methods witr ease at runtime. **0.3.1 or newer required**
+- `HookedMethod` - Hook methods with ease at runtime. **0.3.1 or newer required, earlier versions are broken**
```

Comment:
No need to mention that "earlier versions are broken", unless we also want to mention that older versions of Mono.Cecil don't work due to API changes.

Reply:
HookedMethod forces you to use 0.10.0 AFAIK.

Reply:
So does MonoMod. Again, no need to mention that earlier versions of HookedMethod 0.3.1 are broken - a simple "0.3.1 or newer required" is required, just like with the Mono.Cecil recommendation.

Reply:
Changed this